app/Resources/assets/js/admin.js

```diff
@@ -0,0 +1,69 @@
+var $ = require('jquery');
```

Comment:
What about putting the `assets` directory at the root of the project? It's a bit more "flex" like